config/environments/test.rb

```diff
@@ -1,8 +1,9 @@
 Rails.application.configure do
+  config.assets.raise_runtime_errors = true
   config.cache_classes = true
   config.eager_load = false
   config.serve_static_files   = true
-  config.static_cache_control = "public, max-age=3600"
+  config.static_cache_control = 'public, max-age=3600'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.